plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerTest.java

```diff
@@ -566,19 +566,14 @@ public void warningExistsTest() {
     public void checkVersionCompatibilityNullTest() {
         pm.setJenkinsVersion(null);
 
-        ByteArrayOutputStream output = new ByteArrayOutputStream();
-        System.setOut(new PrintStream(output));
-
         Plugin plugin1 = new Plugin("plugin1", "1.0", null, null);
         plugin1.setJenkinsVersion("2.121.2");
 
         Plugin plugin2 = new Plugin("plugin2", "2.1.1", null, null);
         plugin2.setJenkinsVersion("1.609.3");
 
-        List<Plugin> pluginsToDownload = new ArrayList<>(Arrays.asList(plugin1, plugin2));
-        pm.checkVersionCompatibility(pluginsToDownload);
-
-        assertEquals("", output.toString());
+        //check passes if no exception is thrown
+        pm.checkVersionCompatibility(Arrays.asList(plugin1, plugin2));
```

Comment:
would it be better to add an assertion?

https://junit.org/junit5/docs/current/api/org.junit.jupiter.api/org/junit/jupiter/api/Assertions.html#assertDoesNotThrow(org.junit.jupiter.api.function.Executable,java.lang.String)

Reply:
My idea is to first clean up the tests and then replace JUnit 4 with JUnit Jupiter so that we can use the `assertDoesNotThrow` assertion.

Reply:
How do you feel about this?

Reply:
sure